### PR TITLE
docs/tutorials: add a basic network config

### PR DESCRIPTION
**Pull request by vbatts**
Signed-off-by: Vincent Batts <vbatts@hashbangbash.com>

### CODE REVIEW

**Comment by mheon** [comment]
Should we tell them to install our CNI config instead? We have a good default living in the repo (in contrib/cni I think?)

**Comment by vbatts** [comment]
sure? I'm trying to package cri-o, buildah, and podman for another distro, and files like the cni config make for messy deps, i feel.

**Comment by mheon** [comment]
I think you're safe to include the Podman one in the Podman package. It's set to be (fairly) low priority by default, so installing the CRI-O one shouldn't conflict with it. We explicitly request the network by name if `/etc/containers/libpod.conf` exists and is the default from this repo.

File: docs/tutorials/podman_tutorial.md [hunk 1]
```diff
@@ -81,6 +81,17 @@ $ ./build.sh
 $ sudo mkdir -p /usr/libexec/cni
 $ sudo cp bin/* /usr/libexec/cni
 ```
+#### Installing CNI config
+Add a most basic network config
+```console
+mkdir -p /etc/cni/net.d
+cat > /etc/cni/net.d/99-loopback.conf <<EOF
```
Comment (baude) [inline docs/tutorials/podman_tutorial.md]: My understanding of CNI from Dan Williams is that loopback device declaration is wholly unnecessary.  I think Matt is on the right track with using the default one.

**Comment by rhatdan** [comment]
@vbatts Whats going on with this PR, do you agree with @mheon ?

**Comment by mheon** [comment]
LGTM

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Pull request merged**